src/backend/access/appendonly/test/aomd_test.c

```diff
@@ -15,6 +15,8 @@ static bool file_present[MAX_SEGNO_FILES];
 static int num_unlink_called = 0;
 static bool unlink_passing = true;
 
+int __wrap_do_truncate(const char *path);
```

Comment:
I suggest to remove this redundant declaration

Reply:
It will lead to warning 
```
aomd_test.c:68:1: warning: no previous prototype for ‘__wrap_do_truncate’ [-Wmissing-prototypes]
```
so better to leave it.